### DISCUSSION TITLE
How to setup to use Microsoft Graph API?

### DISCUSSION THREAD

**Discussion by ronald-heijn-buys** [grafana/grafana-infinity-datasource#561] [Q&A]
Hello,

I would like to use this plugin with the Microsoft Graph. I was looking for some more documentation about how to do this but I couldn't find it.

Can somebody help me how to set this up based on an App Registration from Azure AD?

**Comment by robinruegg** [comment]
https://docs.logrhythm.com/OCbeats/docs/configure-microsoft-graph-api
